src/cmds/import:data.ts

```diff
@@ -87,6 +87,8 @@ export async function handler(argv: {
         : `⚠ No import ${entityName[source]}(s) data generated!`;
 
     console.log(targetsMessage);
+
+    return res.fileName
```

Comment:
This handler function is a special yargs function, what we want is for this `handler` function to be able to call another function or 5 in a row to achieve what we need. So what is expected here is that everything inside handler is a *new* function called something like generateOrgData() and this new function returns a fileName etc